### DISCUSSION TITLE
Life Predictor Poem

### DISCUSSION THREAD

**Discussion by sengiv** [gen-so/Genso.Astrology#160] [General]
Tis Fate Then?
No, my friend, it’s not the case,
The planets move through endless space.
They care not for your joy or strife,
Their dance does not decide your life.

Swami Rama, wise and free,
Wrote of dodged karma playfully.
If fate were sealed, how could it be,
That masters shape their destiny?

Weather of the Stars
Thunderstorms may come your way,
But you decide to go or stay.
When skies turn RED, avoid the fight,
But GREEN, it shines—a guiding light.

The storms of life may rage and roar,
Yet calm awaits on peaceful shores.
Choose wisely when to brave the sea,
And sail through life with harmony.

Astro Umbrella
Gems and metals, ancient tools,
Defy the planets, bend their rules.
Before RED strikes, your shield prepare,
And never let it leave you bare.

Gandhi’s tale, a lesson deep,
His father cast his bangle’s keep.
And only then, at journey’s end,
Did life and death in peace ascend.

Fear Not Death
Why fear the day you fade from view?
It’s but a door you’re passing through.
Some go gently, free of dread,
Their GREEN-lit path to peace is led.

Death’s no foe, just time to part,
A fleeting pause, a brand-new start.
To fear it’s like to dread your birth,
When life begins, and joy takes worth.

Winning the Prize
The prize you seek, it may not be,
A chest of gold, but love’s decree.
For wealth is measured by the heart,
In bonds we build, in life’s own art.

Not all will win a lottery’s gain,
For joy wears many forms and names.
The rarest prize is happiness,
Found in life’s sweet, simple bliss.

A Dreamer's Vision
Imagine a world where harm takes flight,
Where accidents fade, and all feels right.
No tragedies darken the human thread,
A life of laughter, with no dread.

In fleeting moments, much is at stake,
One lapse could cause the earth to quake.
Yet dreamers dare to shape the way,
For hope to rise with every day.

"You may say I’m a dreamer," true,
"But not the only one," like you.
Together we’ll envision more,
A brighter world to long adore.

**Comment by agg24ankit** [comment]
@sengiv Are you planning to add Bhrigu nandi nadi, chandra kala nadi, etc method of astrology also.

**Reply by sengiv** [reply]
At the moment no, because I do not have a good reference book on that subject.
But if you wish to add it, please be my guest. I'd be happy to assist you where possible.

**Reply by agg24ankit** [reply]
@sengiv I can provide you the links for the books. Basically bundles of many book which have nadi astrology as one branch of astrology does not answer the question or do correct prediction. 
Here is the link, you can check there are many books available. RG Rao is the authentic one for Bhrigu nandi nadi. Other are chandra kala nadi.  
https://mybapuji.com/pdf-nadi-astrology-books-free-download/

Also, I was trying to run your code as per documentation which you have provided but not able to run it as it is getting failed due to dependencies, seems like that documentation to run the code is not updated that is why I am getting errors. I have expertise in Java but not in .net. Can you please provide me the updated step to run it.

**Reply by sengiv** [reply]
Thank you for the link. 
What do you mean by "_does not answer the question or do correct prediction._" ? Are you saying you've tested the methods in those books and you did not find credibility in real world cases?

Regarding running code. You'll need to specify which code are you trying to run?
The "API" & "Library" project is something I use everyday. And it is running as expected.
Please post the errors you're getting in Issues, with screenshots, we will investigate from there.

**Reply by agg24ankit** [reply]
@sengiv I mean correct prediction generally can't happen as many things are still missing. if you say so then here is my birth detail 24-12-1988, 4:17pm, dehradun, uttarakhand, india. Tell me about my life events. 

I always first match event from my life with any method by Bhrigu system, parashari system, nadi, etc. We can get some few generic answer but not all as actual prediction. The way of using Machine learning algo with existing data is good way to make the prediction better which you have done in your code, thanks for the good work which you have done. You can also use some extra astro bank by saptarishi astrology(https://saptarishisshop.com/product/saptarishis-research-software/?srsltid=AfmBOorSZBjv33-6Nge0lVJvUBDvf35X2y1qOP_kP91i5TiMd7suGIiT)  and lunar astrology(https://lunarastro.com/courses/bhrigu-software-astro-data-bank/)
